# /database/srv/SetDate.srv

uint32 id
string water_date 
---
bool success